sceptre/strides/templates/cloudtrail.yaml

```diff
@@ -0,0 +1,124 @@
+Description: Setup Cloudtrail
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  OperatorEmail:
+    Type: String
+  LogCentralBucket:
+    Type: String
+    Description: The central S3 bucket where AWS sends logs to.
+Resources:
+  LogsCloudtrailLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    Properties:
+      LogGroupName: !Sub '/aws/cloudtrail/${AWS::StackName}.log'
+      RetentionInDays: 90
```

Comment:
This value should be longer... permanent perhaps

Reply:
removed the parameter so it will never expire